packages/syft/src/syft/client/client.py

```diff
@@ -924,13 +932,11 @@ def login(
             _client = _client_cache
 
     if not _client.authed and login_credentials:
-        _client.login(
+        _client = _client.login(
             email=login_credentials.email,
             password=login_credentials.password,
             cache=cache,
         )
-        if not _client.authed:
```

Comment:
How do is a failed login attempt handled?

Reply:
It would return a `SyftError`

```python
_client = ...
if isinstance(_client, SyftError): return _client

return _client
```

in both cases we `return _client` so that check is not needed here